tgintegration/containers/inlineresults.py

```diff
@@ -131,7 +133,11 @@ def _match(self, pattern, getter) -> List:
         return results
 
     def find_results(
-        self, title_pattern=None, description_pattern=None, message_pattern=None, url_pattern=None,
+        self,
+        title_pattern=None,
+        description_pattern=None,
+        message_pattern=None,
+        url_pattern=None,
     ) -> Set[InlineResult]:
 
         # TODO:
```

Comment:
Still to do?

Reply:
Yep, I haven't touched the inline stuff yet.

- [ ] touch the inline stuff 😄